elements/bulbs-slick-slideshow/bulbs-slick-slideshow.js

```diff
@@ -109,10 +113,15 @@ class BulbsSlickSlideshow extends BulbsHTMLElement {
 
   slideshowChanged (event, slickObject, currentSlide) {
     window.location.hash = currentSlide;
-    window.onionan.trackPageView(true);
+    this.trackPageView();
     this.enableDisableNav(this.slides, currentSlide);
   }
 
+  trackPageView () {
+    this.analyticsManager.setDimension('navigation_method','slideshow');
+    this.analyticsManager.trackPageView(false);
```

Comment:
aligning this with clickventure - this `false` param pertains to `freshPage` param which is called with `sendQuantcastPixel` & `sendComscorePixel`

should keep this consistent, since if clickventure isn't considering another frame to be a new page a slideshow shouldn't either.